### PR TITLE
Improve Tree str and html

### LINKED ISSUE #1269: HTML repr for Tree
Similar to tree sequence. __str__ should also give same info.

### PR DESCRIPTION
**Pull request by benjeffery**
Fixes #1269 

Examples:
Screenshot from 2021-04-08 17-07-04
Screenshot from 2021-04-08 17-06-45

### CODE REVIEW

**Comment by benjeffery** [comment]
The original issue suggested average arity - but I couldn't see a quick way to get number of edges in the tree to do edges/nodes?

File: python/tests/test_highlevel.py [hunk 1]
```diff
@@ -2145,10 +2146,38 @@ def test_complex_mutations(self):
...
+            ║Total Branch Length│10.427971║
+            ╚═══════════════════╧═════════╝
+        """
+            )[1:]
+        )
+
+    def test_html_repr(self, ts_fixture):
+        html = ts_fixture.first()._repr_html_()
+        assert len(html) > 1900
+        assert "<tr><td>Total Branch Length</td><td>10.427971</td></tr>" in html
```
Comment (jeromekelleher) [inline python/tests/test_highlevel.py:2180]: Be nice to test if this is valid XML  as well, just pushing it through one of the parsers will be fine.

**Review by jeromekelleher — APPROVED** [review]
LGTM, one small suggestion for improving tests.

We can add other stuff  to the table over time if we want, but this is a great start.

File: python/tests/test_highlevel.py (hunk 1 above)
Comment (benjeffery) [reply python/tests/test_highlevel.py:2185]: Done

**Pull request merged**